docs/actions.md

```diff
@@ -5,28 +5,30 @@ Use actions to update the state.
 ```jsx
 app({
   state: "Hi.",
-  view: (state, actions) => (
-    <h1 onclick={actions.ucase}>{state}</h1>
-  ),
+  view: (state, actions) =>
+    <h1 onclick={actions.ucase}>
+      {state}
+    </h1>,
   actions: {
     ucase: state => state.toUpperCase()
   }
 })
 ```
 
-To be able to update the state, an action must return a new state or a part of it.
+An action must return a new state or a part of it in order to update the state.
```

Comment:
Could you add an example which shows how to update a state object?  This would more clearly illustrate how objects get merged.

For example, show how code like this:
```jsx
app({
  state: {
    counter: 0,
    other: "Value"
  },
  actions: {
    up: state => ({ counter: state.counter + 1})
  }
})
```
results in a state object like this:
```js
{
  counter: 1,
  other: "Value"
}
```